modules/models/src/main/java/org/eclipse/xpanse/modules/models/resource/DeployVariable.java

```diff
@@ -51,7 +52,6 @@ public class DeployVariable {
     @Schema(description = "Validator of the variable")
     private String validator;
 
-    @NotNull
     @Schema(description = "Sensitive scope of variable storage")
-    private SensitiveScope scope;
+    private SensitiveScope scope = SensitiveScope.NONE;
```

Comment:
please also call the field name `sensitiveScope` instead of just `scope`.

Also, please see if this default is correctly displayed in the swagger UI / open API doc, if not, please add it to the description.

Reply:
ok

Reply:
Default values ​​are displayed correctly